tests/conftest.py

```diff
@@ -223,7 +223,7 @@ def uploadAllContracts(self):
                 if name == 'controller': continue
                 # remove this and Interfaces.sol when https://github.com/ethereum/solidity/issues/2665 is fixed
                 if name == 'Interfaces': continue
-                contractsToDelegate = ['orders','tradingEscapeHatch']
+                contractsToDelegate = ['tradingEscapeHatch']
```

Comment:
Orders should still be delegated, so this change should be rolled back.

Reply:
I updated the contract declaration in Orders.sol to `contract Orders is DelegationTarget, Controlled {`.  However, when I tried to change this line to `contractsToDelegate = ['tradingEscapeHatch']`, every script started failing with errors like this:

```
request = <FixtureRequest for <Function 'test_cancelBid'>>

    def fill(request):
        item = request._pyfuncitem
        fixturenames = item.fixturenames
        argnames = item._fixtureinfo.argnames
    
        for fname in fixturenames:
            if fname not in item.funcargs and fname not in argnames:
>               item.funcargs[fname] = request.getfixturevalue(fname)

/usr/local/lib/python2.7/site-packages/pytest_lazyfixture.py:35: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
conftest.py:338: in sessionFixture
    return ContractsFixture()
conftest.py:155: in __init__
    self.initializeAllContracts()
conftest.py:252: in initializeAllContracts
    self.contracts[contractName].initialize(self.controller.address)
/usr/local/lib/python2.7/site-packages/ethereum/tools/tester.py:97: in kall
    startgas=kwargs.get('startgas', STARTGAS)
/usr/local/lib/python2.7/site-packages/ethereum/tools/tester.py:147: in tx
    o = self.direct_tx(transaction)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <ethereum.tools.tester.Chain object at 0x7f1986be0910>, transaction = <Transaction(08fa)>

    def direct_tx(self, transaction):
        self.last_tx, self.last_sender = transaction, None
        success, output = apply_transaction(self.head_state, transaction)
        self.block.transactions.append(transaction)
        if not success:
>           raise TransactionFailed()
E           TransactionFailed

/usr/local/lib/python2.7/site-packages/ethereum/tools/tester.py:140: TransactionFailed
```

Reverting this line back to `contractsToDelegate = ['tradingEscapeHatch']` caused the errors stop occurring.